2017-02-20 14:05:52,795-ERROR-_calcImg:..\..\..\src\opencv\modules\imgproc\src\shapedescr.cpp:799: error: (-201) Number of points should be >= 5
Traceback (most recent call last):
  File "D:\MyProjects\WorkProject\opencv4fiber\geomety\GUI\model\modecv.py", line 95, in _calcImg
    self.eresults = self.classify.find(img)
  File "D:\MyProjects\WorkProject\opencv4fiber\geomety\pattern\classify.py", line 201, in find
    coreResult = PickCircle().run(coreimg)
  File "D:\MyProjects\WorkProject\opencv4fiber\geomety\pattern\pickmethod.py", line 34, in run
    ellipse = cv2.fitEllipse(mergedpoints)
error: ..\..\..\src\opencv\modules\imgproc\src\shapedescr.cpp:799: error: (-201) Number of points should be >= 5

2017-02-20 15:24:45,413-ERROR-_calcImg:
Traceback (most recent call last):
  File "D:\MyProjects\WorkProject\opencv4fiber\geomety\GUI\model\modecv.py", line 95, in _calcImg
    self.eresults = self.classify.find(img)
  File "D:\MyProjects\WorkProject\opencv4fiber\geomety\pattern\classify.py", line 201, in find
    coreResult = PickCircle().run(coreimg)
  File "D:\MyProjects\WorkProject\opencv4fiber\geomety\pattern\pickmethod.py", line 29, in run
    raise ClassCoreError
ClassCoreError
2017-02-23 10:22:44,072-ERROR-_calcImg:
Traceback (most recent call last):
  File "D:\MyProjects\WorkProject\opencv4fiber\geomety\GUI\model\modecv.py", line 95, in _calcImg
    self.eresults = self.classify.find(img)
  File "D:\MyProjects\WorkProject\opencv4fiber\geomety\pattern\classify.py", line 201, in find
    coreResult = PickCircle().run(coreimg)
  File "D:\MyProjects\WorkProject\opencv4fiber\geomety\pattern\pickmethod.py", line 29, in run
    raise ClassCoreError
ClassCoreError
2017-02-23 14:11:10,888-ERROR-_calcImg:
Traceback (most recent call last):
  File "D:\MyProjects\WorkProject\opencv4fiber\geomety\GUI\model\modecv.py", line 95, in _calcImg
    self.eresults = self.classify.find(img)
  File "D:\MyProjects\WorkProject\opencv4fiber\geomety\pattern\classify.py", line 231, in find
    coreResult = PickCircle().run(coreimg)
  File "D:\MyProjects\WorkProject\opencv4fiber\geomety\pattern\pickmethod.py", line 29, in run
    raise ClassCoreError
ClassCoreError
